planning/behavior_path_planner/src/scene_module/goal_planner/goal_planner_module.cpp

```diff
@@ -860,32 +852,26 @@ void GoalPlannerModule::updateSteeringFactor(
     pose, distance, SteeringFactor::GOAL_PLANNER, steering_factor_direction, type, "");
 }
 
+// NOTE: this function returns true once it returns true. Because selectSafePullOverPath() will
+// not select new path.
```

Comment:
:heart:

Reply:
```suggestion
// NOTE: Once this function returns true, it will continue to return true thereafter. Because selectSafePullOverPath() will not select new path.
```